### DISCUSSION TITLE
Setting up LetsEncrypt - noob instructions

### DISCUSSION THREAD

**Discussion by guiltlab** [AzuraCast/AzuraCast#5153] [General]
Hey everyone!

I just managed to set up LetsEncryt and thought I'd post some info about it here.

1. follow instructions at https://docs.azuracast.com/en/administration/ssl-and-lets-encrypt
`cd /var/azuracast`
`./docker.sh update-self`
`./docker.sh letsencrypt-create`
2. open port 443 for your server/router ! radio broadcast ports (e.g. port 8000) should already be open if you used Azuracast in http before, if not be sure to open them otherwise the radio player will not work.
3. In Administration > System settings > Settings, make sure to enable "Web Proxy for Radio" option!!
4. In Administration > System settings > Security & Privacy, enable "Always Use HTTPS"

This may all sound very obvious, but the reason I made this post is that I did not do step 3 and my player did not have any sound, despite my site being https. I searched for the reason for a long time and thought it had something to do with closed ports or something. So maybe this can save someone some time ^^ 

Perhaps it should even be added to instructions at https://docs.azuracast.com/en/administration/ssl-and-lets-encrypt that you have to enable the Web proxy for radio when switching to HTTPS ?

Cheers!

**Comment by perrymartins** [comment]
Please how do I open port 443, I am having challenges with my Let's Encrypt insallation.

**Comment by SpeedyCharly** [comment]
This here noob couldn't get LetsEncrypt to set up properly, even with this recipe, so I totally flushed my installation and redid from scratch, installing last stable and saying YES to the LetsEncrypt prompt.  Still wouldn't give me sound in https mode... So, despairing, I did an update, switching to Rolling release and TADAH!!!  everything worked like a charm after that.  

Added bonus, my small server doesn't go into buffer and/or CPU overruns anymore.... this camper is happy as a pig in you know what!!!
